FAQ.md

```diff
@@ -34,6 +34,14 @@ containers (i.e. libvirtd).
 But in case that you have issues, please
 [file a bug](https://github.com/kubevirt/kubevirt/issues).
 
+## What's the difference between KubeVirt and Kata containers?
+
+[Kata containers](https://katacontainers.io/) allow you to run containers inside virtual machines.  
+KubeVirt allows you to run full virtual machines on Kubernetes with the same as ease as running container workloads.  
```

Comment:
I think you should also add that KubeVirt lets you run VM's alongside containers on the same platform.

Reply:
Done